test/validators/movieList.test.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const Joi = require('joi');
+
+const MovieListValidator = require('../../lib/validators/movieList');
+
+describe('movie validator', () => {
+
+  describe('title', () => {
+
+    it('is optional', () => {
+      const payload = {
+        title: 'foo'
```

Comment:
By providing the `title` param, you're not really testing that it's optional. You should omit the param and ensure that there still is no error.